npm/webpack-preprocessor/lib/cross-origin-callback-compile.ts

```diff
@@ -0,0 +1,76 @@
+import _ from 'lodash'
+import Debug from 'debug'
+import * as path from 'path'
+import webpack from 'webpack'
+import { CrossOriginCallbackStoreFile } from './cross-origin-callback-store'
+
+const VirtualModulesPlugin = require('webpack-virtual-modules')
+
+const debug = Debug('cypress:webpack')
+
+const getConfig = ({ files, originalFilePath }) => {
+  const dir = path.dirname(originalFilePath)
+
+  return files.reduce((memo, file) => {
+    const { inputFileName, source } = file
+    const inputPath = path.join(dir, inputFileName)
+
+    memo.entry[inputFileName] = inputPath
+    memo.virtualConfig[inputPath] = source
+
+    return memo
+  }, { entry: {}, virtualConfig: {} })
+}
+
+const getWebpackOptions = ({ webpackOptions, entry, virtualConfig, outputDir }) => {
```

Comment:
```suggestion
const getWebpackOptions = ({ webpackOptions, entry, virtualConfig, outputDir }: {
  webpackOptions: webpack.Configuration
  entry: { [key: string]: string }
  virtualConfig: { [key: string]: string }
  outputDir: string
}) => {
```

Reply:
Addressed in https://github.com/cypress-io/cypress/pull/23283/commits/a3826c68871558f1c1808fc8ad17bc63d532ad3d